its/src/test/java/com/sonar/it/csharp/IncrementalAnalysisTest.java

```diff
@@ -110,7 +111,11 @@ public void incrementalPrAnalysis_cacheAvailableChangesDone_issuesReportedForCha
     assertThat(endStepResults.getLogs()).doesNotContain("Adding normal issue S1134: " + unchanged2Path);
     assertThat(endStepResults.getLogs()).contains("Adding normal issue S1134: " + withChangesPath);
     assertThat(endStepResults.getLogs()).contains("Adding normal issue S1134: " + fileToBeAddedPath);
-    List<Issues.Issue> allIssues = TestUtils.getIssues(ORCHESTRATOR, PROJECT, "42");
+    List<Issues.Issue> allIssues = TestUtils
+        .getIssues(ORCHESTRATOR, PROJECT, "42")
+        .stream()
+        .filter(x -> x.getRule().startsWith("csharpsquid:S1134"))
```

Comment:
I guess this should be equals, rather than startsWith?

Reply:
Changed it to `equals()`.